### DISCUSSION TITLE
Old server LGSM to new server docker Plaword

### DISCUSSION THREAD

**Discussion by god81000** [thijsvanloef/palworld-server-docker#566] [Q&A]
Good morning, 
Thank you already for this github project which allowed me to have a palword server on my docker. But I have a question or rather a problem, before having this problem I had another server where I had made a palwird server with lgsm we did a lot of things in this server and I made a backup, but I don't know if my backup of my old server can be integrated into the container. I tried to do a backup restore with the folder of my old server but there is nothing every time this happens, has anyone already had this problem and if they were able to solve it?

backup of my old serv :

![image](https://github.com/thijsvanloef/palworld-server-docker/assets/75736124/054de9fe-f42c-4444-8e6f-135a8e6b68ca)

**Comment by thijsvanloef** [comment]
You might want to try the following steps:

1. Copy the save from your old dedicated server to your new dedicated server.
2. In the `PalServer\Pal\Saved\Config\LinuxServer\GameUserSettings.ini` file of the new server, change the `DedicatedServerName `to match your save's folder name. For example, if your save's folder name is `2E85FD38BAA792EB1D4C09386F3A3CDA`, the `DedicatedServerName `changes to `DedicatedServerName=2E85FD38BAA792EB1D4C09386F3A3CDA`.
3. Delete the entire new server save at `PalServer\Pal\Saved\SaveGames\0\<your_save_here>`, and replace it with the folder from the old server.
 4. Restart the new server

Please make sure you have your old saved backed up before attempting this. 
I do not have experience with migrating from LinuxGSM to my docker container, and am not sure if this will work.

**Comment by god81000** [comment]
Thank you for everything, it's going very well.

**Comment by thijsvanloef** [comment]
Since this is not a direct issue with the image, I am converting it to a discussion
